bciers/apps/administration/app/components/operators/OperatorPage.tsx

```diff
@@ -1,4 +1,3 @@
-import { notFound } from "next/navigation";
 import OperatorForm from "./OperatorForm";
 import { operatorSchema } from "../../data/jsonSchema/operator";
```

Comment:
Tiniest of nits but I think moving up multiple directories like this in imports makes the code less readable. We can import from `apps/administration/data/jsonSchema/operator` and still avoid the circular import issue when using `@/administration` and the next person to read this file will know exactly where that import is coming from :)

Reply:
@marcellmueller 
Totally agree! 

I didn't realize we could avoid the circular import with that technique- thanks :)

Reply:
To be clear I'm not sure if that is okay for shared imports outside of the project 🤔